util/calcrewards.py

```diff
@@ -172,6 +172,8 @@ def _calcRewardsUsd(
                 if Sij == 0 or Cj == 0:
                     continue
                 RF_ij = Sij * Cj  # main formula!
+                if RF_ij < 0.001:
+                    continue
```

Comment:
Filtering won't work well here, since the units are a bit hard to reason about, being it's (stake in USD) * (volume in USD). 

I recommend instead to delete your change, and instead change "normalize and scale" part to something like the following.

1. Normalize
2. Then filter on normalized values, where units are known (a percentage)
3. Then normalize again, and scale

That is:
```python
    # normalize rewards (don't scale)
    for chainID in chainIDs:
        for LP_addr, reward in rewardsperlp[chainID].items():
            rewardsperlp[chainID][LP_addr] = reward / tot_rewards

    #filter out any normalized reward that's less than 0.001% (=0.00001)
    for chainID in chainIDs:
        for LP_addr, reward in rewardsperlp[chainID].items():
            if rewardsperlp[chainID][LP_addr] < 0.00001:
                del rewardsperlp[chainID][LP_addr]

    # normalize again, *and* scale rewards
    for chainID in chainIDs:
        for LP_addr, reward in rewardsperlp[chainID].items():
            rewardsperlp[chainID][LP_addr] = reward / tot_rewards * TOKEN_avail
```

Reply:
Whups, yes, do that.